crates/types/src/nodes_config.rs

```diff
@@ -339,58 +339,63 @@ impl Versioned for NodesConfiguration {
+    /// A node can never transition back to `Provisioning` once it has transitioned out of it.
     ///
-    /// or all intents and purposes, this is equivalent to a `ReadOnly` state, except that it's
-    /// excluded from nodeset generation. The difference
-    /// between this and `ReadOnly` is that if a node is in Provisioning state, we are confident
-    /// that it has not been added to nodesets and we can safely remove it from the cluster without
-    /// checking the log-chain or trim points of loglets. Note that if this node happens to be in a
-    /// nodeset (although control plan shouldn't add it) spread selectors will not write any data
-    /// to it until they observe a state transition to ReadWrite. This behaviour matches `ReadOnly`
-    /// state.
+    /// For all intents and purposes, this is equivalent to a `ReadOnly` state, except that it's
+    /// excluded from nodeset generation. The difference between this and `ReadOnly` is that, if a
+    /// node is in Provisioning state, we are confident that it has not been added to node sets, and
+    /// we can safely remove it from the cluster without checking the log-chain or trim points of
+    /// loglets. Note that if this node happens to be in a nodeset (although control plane shouldn't
+    /// add it), spread selectors will not write any data to it until they observe a state
+    /// transition to `ReadWrite`. This behaviour matches the `ReadOnly` state.
     ///
-    /// can read from: yes (likely to not have data, but it might if it transitioned into RW)
-    /// can write to: yes - but excluded from new nodesets. If you see it in a nodeset, try writing to it.
+    /// - can read from: **yes** (likely to not have data, but it might if it transitioned into `ReadWrite`)
+    /// - can write to: **yes** but excluded from new node sets; if you see it in a nodeset, try writing to it
+    /// - new node sets: **excluded**
     #[default]
     Provisioning,
+
     // [authoritative empty]
-    /// Node's storage is not expected to be accessed in reads nor write. The node is not
-    /// considered as part of the replicated log cluster. Node can be safely decommissioned.
+    /// Node's storage is not expected to be accessed for reads or writes. The node is not
+    /// considered as part of the replicated log cluster, and can be safely decommissioned.
     ///
     /// The node can never transition out of `Disabled` after it has transitioned into it.
     ///
-    /// should read from: no
-    /// can write to: no
+    /// - should read from: **no**
+    /// - can write to: **no**
     Disabled,
+
     // [authoritative]
-    /// Node is not picked in new write sets, but it may still accept writes on existing nodeset
-    /// and it's included in critical metadata updates (seal, release, etc.)
+    /// Node is not picked for new write sets, but it may still accept writes on existing node sets,
```

Comment:
Shouldn't this be `nodesets` as agreed

```suggestion
    /// Node is not picked for new write sets, but it may still accept writes on existing nodesets,
```